chukwuemekachm/utility_manager_web
# Prof: Ensures api calls work as expected with Cookie infrastructure

#### What does this PR achieve?
- adds withCredentials to Axios instance
- uses sample dashboard skeleton to test that everything is working as
expected
- ensures that API calls are made successfully
- redirects user to dashboard when `Sign up` link is clicked on from email

#### Any background context?
- I used a sample dashboard page for testing that this works. You can change everything in subsequent tasks

#### Manual Testing Steps
- Open app and login
- You should be redirected to dashboard page with your info showing in bullet points

#### Screenshots:
<img width="1440" alt="Screenshot 2020-01-06 at 6 45 49 PM" src="https://user-images.githubusercontent.com/38565349/71836669-ca082780-30b4-11ea-8125-4d0590d09038.png">

<img width="1440" alt="Screenshot 2020-01-05 at 8 06 27 AM" src="https://user-images.githubusercontent.com/38565349/71786310-60cbda00-300a-11ea-9105-d26f9f435fb0.png">
<img width="1440" alt="Screenshot 2020-01-05 at 8 06 19 AM" src="https://user-images.githubusercontent.com/38565349/71786311-60cbda00-300a-11ea-8278-fcc74642b98b.png">

#### Relevant Trello Tickets
- [Ensure that API call works as expected](https://trello.com/c/8XrOBSRU/4-ensure-api-call-works-as-expected)

# This is currently Blocked by [Endpoint to view profile](https://trello.com/c/95RNdvPo/59-d-1-endpoint-to-allow-user-view-profile)
